homeassistant/components/light/__init__.py

```diff
@@ -12,7 +12,8 @@
 
 import voluptuous as vol
 
-from homeassistant.components import group
+from homeassistant.components.group import \
+    ENTITY_ID_FORMAT as GROUP_ENTITY_ID_FORMAT
```

Comment:
By importing `components.group` into the light namespace I was unable to import `light.group` (the Group Light platform) in the tests, so I now import the "ENTITY_ID_FORMAT" directly with a `GROUP_*` prefix. Let me know if there's a better way.

Reply:
that's fine.